platformio.ini

```diff
@@ -97,6 +97,16 @@ build_flags =
 build_partitions = min_spiffs.csv
 build_partitions_debug = min_spiffs_debug.csv
 
+# Used for LoRaWAN builds
+# https://github.com/mcci-catena/arduino-lmic#selecting-the-lorawan-region-configuration
+lora_lib =
+  MCCI LoRaWAN LMIC library
+  aparcar/CayenneLPP@^1.3.0
+lora_build_flags =
+  -D ARDUINO_LMIC_PROJECT_CONFIG_H_SUPPRESS
+  -D hal_init=LMICHAL_init ; Workaround mcci arduino-lmic bug 714 on esp32
+  -D CFG_us915        ; USA 915Mhz
```

Comment:
Can this be configured at run time?

Reply:
nah. The country is all compile-time definitions:
https://github.com/mcci-catena/arduino-lmic#selecting-the-lorawan-region-configuration

Reply:
Guess we will need lots of builds if LoRa takes off ;-)